components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -280,16 +280,18 @@ func Run(options ...RunOption) {
 		if !cfg.isHeadless() {
 			go startAnalyze(ctx, cfg, gitpodConfigService, topService, gitpodService)
 		}
-		_, gitpodHost, err := cfg.GitpodAPIEndpoint()
-		if err != nil {
-			log.WithError(err).Error("grpc metrics: failed to parse gitpod host")
-		} else {
-			metricsReporter = metrics.NewGrpcMetricsReporter(gitpodHost)
-			if err := supervisorMetrics.Register(metricsReporter.Registry); err != nil {
-				log.WithError(err).Error("could not register supervisor metrics")
-			}
-			if err := gitpodService.RegisterMetrics(metricsReporter.Registry); err != nil {
-				log.WithError(err).Error("could not register public api metrics")
+		if !strings.HasPrefix("ws-ephemeral", cfg.WorkspaceClusterHost) {
```

Comment:
@utam0k @kylos101 does it make sense?

Reply:
/hold for the answer to the question above.

Reply:
I would check if the cluster name contains `ephemeral` instead. Not everyone uses a ws-ephemeral prefix when naming  a cluster but it should contain ephemeral.

Reply:
fixed, @Furisto could you have a look again?